### PR TITLE
OSDOCS-4073: Release notes for dynamic plug-in

### PR DESCRIPTION
**Pull request by opayne1**
[OSDOCS-4073](https://issues.redhat.com//browse/OSDOCS-4073): Release notes for dynamic plug-in

Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4073

Link to docs preview:
https://54387--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-dynamic-plug-in-updates

QE review:
- [x] QE has approved this change.

Additional information:
**Note for reviewer: The build will fix after main PR merges.**

### CODE REVIEW

**Comment by opayne1** [comment]
Build is failing because https://github.com/openshift/openshift-docs/pull/53103 has not merged yet.

**Review by spadgett — APPROVED** [review]
lgtm

@alimobrem fyi

**Comment by opayne1** [comment]
@yapei Could you provide a QE review of this release note for the dynamic plug-in docs I am wrapping up? Thank you! 😄

**Comment by yapei** [comment]
looks good to me

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -164,6 +164,19 @@ The `console-operator` now scans all nodes and builds a set of all architecture
 
 For more information about multi-architechture compute machines, see xref:../post_installation_configuration/multi-architecture-configuration.adoc[Configuring a multi-architecture compute machine on an OpenShift cluster].
 
+[id="ocp-4-12-dynamic-plug-in-updates"]
+===== Dynamic plug-in generally available
+
+This feature was previously introduced as a Technology Preview in {product-title} 4.10 and is now generally available in {product-title} 4.12. With the dynamic plug-in you can build high quality and unique user experiences natively in the web console. You can:
```
Comment (mburke5678) [inline release_notes/ocp-4-12-release-notes.adoc:170]: Do we need a comma here? _With the dynamic plug-in,_
I probably would put one; but I tend to over-comma things. Up to you

**Comment by mburke5678** [comment]
@opayne1 Without seeing the preview, I have one nit. Otherwise LGTM

**Review by JoeAldinger — COMMENTED** [review]
/Lgtm

**Review by jldohmann — APPROVED** [review]
lgtm

**Pull request merged**